### PR TITLE
WS-1025: added relationships discoveryMode option

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

File: model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java [hunk 1]
```diff
@@ -56,23 +63,53 @@ public void setAccuracyMode(AccuracyMode accuracyMode) {
...
+        }
+
+
+        /**
+         * DocumentRequest discovery mode. If the value is {@code true}, then the the endpoint will open relationship
+         * extraction results. If {@code false}, not. If {@code null}, the endpoint will perform default processing.
+         * @param discoveryMode discovery mode
+         * @return this.
+         */
+        public RelationshipsOptions.Builder discoveryMode(Boolean discoveryMode) {
```
Comment (guruxu) [inline model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: no `accuracyMode`?

Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: I'm a bit confused about the usage of the builder. I can certainly expose that also.

Comment (guruxu) [reply model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: Just to be consistent if you are to use fluent.

Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: fixed.

Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: oops, would be npe in equals. Not ready to merge.

Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/RelationshipsOptions.java:113]: Done. PR is ready.

**Pull request merged**